Makefile

```diff
@@ -13,9 +13,9 @@ build-app:
 build-test:
 	docker build -f infra/python-backend-test.Dockerfile -t containres-test .
 
-.PHONY: build-webapp
-build-webapp:
-	docker build -f infra/webapp.Dockerfile -t  containres-webapp webapp
+#.PHONY: build-webapp
```

Comment:
in this iteration of the build this is merged in with the python webapp backend to facilitate the current CORS setup of the app